action.yml

```diff
@@ -47,6 +47,10 @@ inputs:
     description: 'Indicates if release notes should be automatically generated.'
     required: false
     default: 'false'
+  makeLatest:
+    description: 'Indicates if the release should be the "latest" release or not.'
+    required: false
+    default: 'true'
```

Comment:
I think the default should be `legacy` so the action behaves the same as it did before this change by default.